### PR TITLE
Upgrade cfengine to 3.17.0 official release

### PR DESCRIPTION
**Pull request by craigcomstock**

### CODE REVIEW

File: packages/cfengine/build.sh [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 TERMUX_PKG_LICENSE="GPL-3.0"
-TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=3.17.0a1-termux
-TERMUX_PKG_SRCURL=(https://github.com/cfengine/core/archive/ec47889f4bef53c6c5a15add60d34c44c6ef1634.zip
-                   https://github.com/cfengine/masterfiles/archive/83c4e0cf9dddffdd192d49f98a30a9bb705edfc4.zip
-                   https://github.com/cfengine/libntech/archive/10f1112b118b05cf9ccc2aedcf4b1ee09090fcd0.zip)
-TERMUX_PKG_SHA256=(53feaf799db0ac7e9579f7101c1ab87dc9bba7e383f3da1a74828e4a6dc005e1
-                   3ea7759125acc6501c49ba58f98cef9cea598f5da810352eda86e1efa4d19b8e
-                   b0458087ab526b5b73bc9e399e420dcef31d4e649898954346f8bd2594bd9446)
+TERMUX_PKG_MAINTAINER="@cfengine"
```
Comment (kcubeterm) [inline packages/cfengine/build.sh:4]: Why you changed maintainer name ?

Comment (craigcomstock) [reply packages/cfengine/build.sh:4]: I read the commit that changed it and thought it was the right thing to do to state who is the maintainer. :) The maintainer is certainly not `@termux` as I have been doing it for a while now and we, `@cfengine`, are certainly the parties who should be contacted with issues.

Comment (craigcomstock) [reply packages/cfengine/build.sh:4]: https://github.com/termux/termux-packages/commit/37375312b93aecee93d86392b8db6b9ab3359c84

**Comment by craigcomstock** [comment]
converted to draft as I am running through some manual testing :)

File: packages/cfengine/build.sh (hunk 1 above)
Comment (kcubeterm) [reply packages/cfengine/build.sh:4]: Yes, you always update it. I thought you are just any random guy. Glad to know that any official members of cfengine is maintaining it.

**Comment by craigcomstock** [comment]
I went through a few basic things and all looks good. :+1:

**Pull request merged**